homeassistant/components/supla/cover.py

```diff
@@ -43,9 +64,9 @@ def current_cover_position(self):
             return 100 - state["shut"]
         return None
 
-    def set_cover_position(self, **kwargs):
+    async def async_set_cover_position(self, **kwargs):
         """Move the cover to a specific position."""
-        self.action("REVEAL", percentage=kwargs.get(ATTR_POSITION))
+        await self.async_action("REVEAL", percentage=kwargs.get(ATTR_POSITION))
```

Comment:
`ATTR_POSITION` is a required parameter, so we should use `kwargs[ATTR_POSITION]`.